Scripts/ExtractFQDNFromUrlAndEmail/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
```

Comment:
no need to specify change log for new entities

Reply:
So just leave it empty?

Reply:
remove the file completely

Reply:
Done